### PR TITLE
Add viewAs=Instructor to relevant Self-Paced PL links

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adds `?viewAs=Instructor` to relevant links on https://code.org/professional-learning/self-paced.

## Links
Jira ticket: [ACQ-2037](https://codedotorg.atlassian.net/browse/ACQ-2037)

## Testing story
Tested locally

### CODE REVIEW

**Review by hannahbergam — APPROVED** [review]
LGTM

**Review by hannahbergam — APPROVED** [review]
Looks like we're adding them to all the PL links too! LGTM

**Comment by kelbyhawn** [comment]
> Looks like we're adding them to all the PL links too! LGTM

Yep! Dani gave the go-ahead ([Slack](https://codedotorg.slack.com/archives/C0453TUMLE7/p1719594680539209?thread_ts=1719583305.729249&cid=C0453TUMLE7)).

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml [hunk 1]
```diff
@@ -7,7 +7,7 @@
       curriculum: hoc_s(:curriculum_name_all_ai),
       duration: hoc_s(:module_duration_5_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-ai-101"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-ai-101?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml]: This is the self paced PL so shouldn't have this on it

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml [hunk 2]
```diff
@@ -23,8 +23,8 @@
         curriculum: hoc_s(:curriculum_name_how_ai_makes_decisions),
         duration: hoc_s(:module_duration_1_hr),
         prerequisites: hoc_s(:module_prerequisites_none),
-        url_primary: CDO.studio_url("/courses/elementaryai-2024"),
-        url_secondary: CDO.studio_url('/s/k5-ai-data-2024'),
+        url_primary: CDO.studio_url("/courses/elementaryai-2024?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml [hunk 3]
```diff
@@ -36,8 +36,8 @@
         curriculum: hoc_s(:curriculum_name_elementary_game_design),
         duration: hoc_s(:module_duration_1_hr_30_minutes),
         prerequisites: hoc_s(:module_prerequisites_none),
-        url_primary: CDO.studio_url("/courses/3-5gamedesign-2024"),
-        url_secondary: CDO.studio_url('/s/elem-game-design-2024'),
+        url_primary: CDO.studio_url("/courses/3-5gamedesign-2024?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml [hunk 4]
```diff
@@ -49,7 +49,7 @@
         curriculum: hoc_s(:curriculum_name_microbit_maker_modules),
         duration: hoc_s(:module_duration_1_hr_30_minutes),
         prerequisites: hoc_s(:module_prerequisites_none),
-        url_primary: CDO.studio_url("/courses/microbitmakermodule-2024"),
+        url_primary: CDO.studio_url("/courses/microbitmakermodule-2024?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml [hunk 5]
```diff
@@ -65,7 +65,7 @@
       curriculum: hoc_s(:curriculum_name_all_k5),
       duration: hoc_s(:module_duration_2_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-k5-2024"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-k5-2024?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 6]
```diff
@@ -7,7 +7,7 @@
       curriculum: hoc_s(:curriculum_name_all_ai),
       duration: hoc_s(:module_duration_5_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-ai-101"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-ai-101?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 7]
```diff
@@ -23,7 +23,7 @@
         curriculum: hoc_s(:curriculum_name_coding_with_ai),
         duration: hoc_s(:module_duration_2_hrs),
         prerequisites: hoc_s(:module_prerequisites_none),
-        url_primary: CDO.studio_url("/courses/self-paced-pl-coding-with-ai"),
+        url_primary: CDO.studio_url("/courses/self-paced-pl-coding-with-ai?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 8]
```diff
@@ -36,7 +36,7 @@
         curriculum: hoc_s(:curriculum_name_computer_vision),
         duration: hoc_s(:module_duration_1_hr_30_minutes),
         prerequisites: hoc_s(:module_prerequisites_none),
-        url_primary: CDO.studio_url("/courses/self-paced-pl-computer-vision"),
+        url_primary: CDO.studio_url("/courses/self-paced-pl-computer-vision?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 9]
```diff
@@ -52,7 +52,7 @@
       curriculum: hoc_s(:curriculum_name_csp),
       duration: hoc_s(:module_duration_3_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csp"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csp?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 10]
```diff
@@ -64,7 +64,7 @@
       curriculum: hoc_s(:curriculum_name_ai),
       duration: hoc_s(:module_duration_2_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-aiml"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-aiml?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 11]
```diff
@@ -76,7 +76,7 @@
       curriculum: hoc_s(:curriculum_name_maker_microbit),
       duration: hoc_s(:module_duration_3_hrs),
       prerequisites: hoc_s(:module_prerequisites_maker_microbit),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-microbit"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-microbit?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 12]
```diff
@@ -88,7 +88,7 @@
       curriculum: hoc_s(:curriculum_name_csd),
       duration: hoc_s(:module_duration_3_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csd"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csd?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 13]
```diff
@@ -100,7 +100,7 @@
       curriculum: hoc_s(:curriculum_name_maker_cp),
       duration: hoc_s(:module_duration_3_hrs),
       prerequisites: hoc_s(:module_prerequisites_maker_cp),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-physical-computing"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-physical-computing?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 14]
```diff
@@ -112,8 +112,8 @@
       curriculum: hoc_s(:module_label_teaching_animations),
       duration: hoc_s(:module_duration_3_hrs_20_minutes),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit3"),
-      url_secondary: CDO.studio_url("/s/interactive-games-animations"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit3?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 15]
```diff
@@ -124,8 +124,8 @@
       curriculum: hoc_s(:curriculum_name_problem_solving),
       duration: hoc_s(:module_duration_2_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit1"),
-      url_secondary: CDO.studio_url("/s/problem-solving-computing"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit1?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 16]
```diff
@@ -136,8 +136,8 @@
       curriculum: hoc_s(:curriculum_name_data_and_society),
       duration: hoc_s(:module_duration_2_hrs_30_minutes),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit5"),
-      url_secondary: CDO.studio_url("/s/data-society"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit5?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml [hunk 17]
```diff
@@ -148,8 +148,8 @@
       curriculum: hoc_s(:curriculum_name_design_process),
       duration: hoc_s(:module_duration_2_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit4"),
-      url_secondary: CDO.studio_url("/s/design-process"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit4?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_high.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 18]
```diff
@@ -7,7 +7,7 @@
       curriculum: hoc_s(:curriculum_name_all_ai),
       duration: hoc_s(:module_duration_5_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-ai-101"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-ai-101?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 19]
```diff
@@ -23,7 +23,7 @@
         curriculum: hoc_s(:curriculum_name_coding_with_ai),
         duration: hoc_s(:module_duration_2_hrs),
         prerequisites: hoc_s(:module_prerequisites_none),
-        url_primary: CDO.studio_url("/courses/self-paced-pl-coding-with-ai"),
+        url_primary: CDO.studio_url("/courses/self-paced-pl-coding-with-ai?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 20]
```diff
@@ -39,8 +39,8 @@
       curriculum: hoc_s(:module_label_teaching_animations),
       duration: hoc_s(:module_duration_3_hrs_20_minutes),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit3"),
-      url_secondary: CDO.studio_url("/s/interactive-games-animations"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit3?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 21]
```diff
@@ -51,7 +51,7 @@
       curriculum: hoc_s(:curriculum_name_maker_cp),
       duration: hoc_s(:module_duration_3_hrs),
       prerequisites: hoc_s(:module_prerequisites_maker_cp),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-physical-computing"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-physical-computing?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 22]
```diff
@@ -63,8 +63,8 @@
       curriculum: hoc_s(:curriculum_name_csd),
       duration: hoc_s(:module_duration_3_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csd"),
-      url_secondary: CDO.studio_url("/courses/csd?viewAs=Instructor"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csd?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 23]
```diff
@@ -75,8 +75,8 @@
       curriculum: hoc_s(:curriculum_name_problem_solving),
       duration: hoc_s(:module_duration_2_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit1"),
-      url_secondary: CDO.studio_url("/s/problem-solving-computing"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit1?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 24]
```diff
@@ -87,7 +87,7 @@
       curriculum: hoc_s(:curriculum_name_maker_microbit),
       duration: hoc_s(:module_duration_3_hrs),
       prerequisites: hoc_s(:module_prerequisites_maker_microbit),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-microbit"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-microbit?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 25]
```diff
@@ -99,7 +99,7 @@
       curriculum: hoc_s(:curriculum_name_ai),
       duration: hoc_s(:module_duration_2_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-aiml"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-aiml?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 26]
```diff
@@ -111,8 +111,8 @@
       curriculum: hoc_s(:curriculum_name_data_and_society),
       duration: hoc_s(:module_duration_2_hrs_30_minutes),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit5"),
-      url_secondary: CDO.studio_url("/s/data-society"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit5?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 27]
```diff
@@ -123,8 +123,8 @@
       curriculum: hoc_s(:curriculum_name_design_process),
       duration: hoc_s(:module_duration_2_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit4"),
-      url_secondary: CDO.studio_url("/s/design-process"),
+      url_primary: CDO.studio_url("/courses/self-paced-pl-csd-unit4?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml [hunk 28]
```diff
@@ -135,8 +135,8 @@
       curriculum: hoc_s(:curriculum_name_web_development),
       duration: hoc_s(:module_duration_2_hrs),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url('/courses/self-paced-pl-csd-unit2'),
-      url_secondary: CDO.studio_url('/s/web-development'),
+      url_primary: CDO.studio_url('/courses/self-paced-pl-csd-unit2?viewAs=Instructor'),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_middle.haml]: remove

File: pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml [hunk 29]
```diff
@@ -77,7 +77,7 @@
       curriculum: hoc_s(:curriculum_name_csf_02),
       duration: hoc_s(:module_duration_4_hrs_30_mins),
       prerequisites: hoc_s(:module_prerequisites_none),
-      url_primary: CDO.studio_url("/s/k5-onlinepd-2023"),
+      url_primary: CDO.studio_url("/s/k5-onlinepd-2023?viewAs=Instructor"),
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml]: remove here sorry if I missed this one

**Comment by dmcavoy** [comment]
One last one to update then looks good!

**Pull request merged**